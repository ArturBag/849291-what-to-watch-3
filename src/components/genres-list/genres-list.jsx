import React from "react";
import MoviesList from "../movies-list/movies-list.jsx";
import PropTypes from "prop-types";
import { connect } from "react-redux";
import { ActionCreator } from "../../reducer.js";


const genresListItems = [`All genres`, `Comedies`, `Crime`, `Documentary`, `Dramas`,
  `Horror`, `Kids & Family`, `Romance`, `Sci-Fi`, `Thrillers`];

const GenresList = (props) => {

  const { moviesList, activeGenre, onMovieCardTitleClick, onGenreTypeClick } = props;
  console.log(activeGenre)
  const genresListData = genresListItems.map((genre, index) => {


    const activeGenreClass = genre === activeGenre ? `catalog__genres-item--active` : ``;

    return (
      <li key={genre + index} className={`catalog__genres-item ${activeGenreClass}`}>
        <a href="#" className="catalog__genres-link"
          onClick={() => {
            onGenreTypeClick(genre);
          }}
        >
          {genre}
        </a>
      </li>
    );
  });


  return (
    <React.Fragment>
      <h2 className="catalog__title visually-hidden">Catalog</h2>
      <ul className="catalog__genres-list">
        {genresListData}
      </ul>
      <div className="catalog__movies-list">
        {<MoviesList
          moviesList={moviesList}
          onMovieCardTitleClick={onMovieCardTitleClick}
        />}
      </div>
      <div className="catalog__more">
        <button className="catalog__button" type="button">
          Show more
        </button>
      </div>
    </React.Fragment>
  );

};

GenresList.propTypes = {
  activeGenre: PropTypes.string.isRequired,
  moviesList: PropTypes.array.isRequired,
  onMovieCardTitleClick: PropTypes.func.isRequired,
  onGenreTypeClick: PropTypes.func.isRequired,
};

const mapStateToProps = (state) => ({
  activeGenre: state.activeGenre,
});

const mapDispatchToProps = (dispatch) => ({
  onGenreTypeClick(genre) {
    dispatch(ActionCreator.onGenreTypeClick(genre));
  }
});

export { GenresList };

export default connect(mapStateToProps, mapDispatchToProps)(GenresList);
