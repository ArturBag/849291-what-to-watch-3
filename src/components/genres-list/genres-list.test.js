import React from "react";
import renderer from "react-test-renderer";
import {GenresList} from "./genres-list.jsx";
import {Provider} from "react-redux";
import configureStore from "redux-mock-store";

const filmsList = [
  {
    title: `Breaking Bad`,
    genre: `Thriller`,
    issuedDate: 2000,
    ratingCount: 59,
    preview: `../../markup/img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.`,
    rating: 8.9,
    director: `Bernard Harding`,
    starring: [`Bill Murray, Edward Norton, Jude Law, Willem Dafoe and other`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    runTime: {hours: 1, minutes: 22},
    reviews: [
      {
        name: `Kate Muir`,
        reviewText: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
        date: `December 24, 2016`,
        score: 2.9
      },
      {
        name: `Bill Goodykoontz`,
        reviewText: `Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        "The Grand Budapest Hotel" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.`,
        date: `November 18, 2015`,
        score: 8.2
      },
      {
        name: `Amanda Greever`,
        reviewText: `I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.`,
        date: `November 18, 2015`,
        score: 8.4
      },
      {
        name: `Matthew Lickona`,
        reviewText: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        date: `December 20, 2016`,
        score: 6.2
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.6
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.0
      },
    ]
  },
  {
    title: `Pirates`,
    genre: `Drama`,
    issuedDate: 1999,
    ratingCount: 11,
    preview: `../../markup/img/bohemian-rhapsody.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.`,
    rating: 4.5,
    director: `Will Brooks`,
    starring: [`Kymani Burgess, Jaylin Cherry, Jacey Oliver, Conrad Wolf and other`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    reviews: [
      {
        name: `Kate Muir`,
        reviewText: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
        date: `December 24, 2016`,
        score: 2.9
      },
      {
        name: `Bill Goodykoontz`,
        reviewText: `Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        "The Grand Budapest Hotel" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.`,
        date: `November 18, 2015`,
        score: 8.2
      },
      {
        name: `Amanda Greever`,
        reviewText: `I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.`,
        date: `November 18, 2015`,
        score: 8.4
      },
      {
        name: `Matthew Lickona`,
        reviewText: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        date: `December 20, 2016`,
        score: 6.2
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.6
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.0
      },
    ]
  },
  {
    title: `Misfits`,
    genre: `Drama`,
    issuedDate: 1998,
    ratingCount: 24,
    preview: `../../markup/img/macbeth.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.`,
    rating: 6.7,
    director: `Titus Sparks`,
    starring: [`Giovanny Mccullough, Leah Delacruz, Davon Terrell, Romeo Carr and other`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    reviews: [
      {
        name: `Kate Muir`,
        reviewText: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
        date: `December 24, 2016`,
        score: 2.9
      },
      {
        name: `Bill Goodykoontz`,
        reviewText: `Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        "The Grand Budapest Hotel" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.`,
        date: `November 18, 2015`,
        score: 8.2
      },
      {
        name: `Amanda Greever`,
        reviewText: `I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.`,
        date: `November 18, 2015`,
        score: 8.4
      },
      {
        name: `Matthew Lickona`,
        reviewText: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        date: `December 20, 2016`,
        score: 6.2
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.6
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.0
      },
    ]
  },
  {
    title: `Terminal`,
    genre: `Horror`,
    issuedDate: 1992,
    ratingCount: 555,
    preview: `../../markup/img/aviator.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.`,
    rating: 6.9,
    director: `Jeramiah Goldeng`,
    starring: [`Nasir Castaneda, Ariel Brooks, Lucia Compton and other`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    reviews: [
      {
        name: `Kate Muir`,
        reviewText: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
        date: `December 24, 2016`,
        score: 2.9
      },
      {
        name: `Bill Goodykoontz`,
        reviewText: `Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        "The Grand Budapest Hotel" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.`,
        date: `November 18, 2015`,
        score: 8.2
      },
      {
        name: `Amanda Greever`,
        reviewText: `I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.`,
        date: `November 18, 2015`,
        score: 8.4
      },
      {
        name: `Matthew Lickona`,
        reviewText: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        date: `December 20, 2016`,
        score: 6.2
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.6
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.0
      },
    ]
  },
  {
    title: `Star Wars`,
    genre: `Documentary`,
    issuedDate: 2002,
    ratingCount: 242,
    preview: `../../markup/img/we-need-to-talk-about-kevin.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.`,
    rating: 8.3,
    director: `Roy Boyle`,
    starring: [`Molly Reyes, Edward Norton, Mareli Moore, Makenna Henderson and other`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    reviews: [
      {
        name: `Kate Muir`,
        reviewText: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
        date: `December 24, 2016`,
        score: 2.9
      },
      {
        name: `Bill Goodykoontz`,
        reviewText: `Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        "The Grand Budapest Hotel" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.`,
        date: `November 18, 2015`,
        score: 8.2
      },
      {
        name: `Amanda Greever`,
        reviewText: `I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.`,
        date: `November 18, 2015`,
        score: 8.4
      },
      {
        name: `Matthew Lickona`,
        reviewText: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        date: `December 20, 2016`,
        score: 6.2
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.6
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.0
      },
    ]
  },
  {
    title: `Forsage`,
    genre: `Crime`,
    issuedDate: 2001,
    ratingCount: 243,
    preview: `../../markup/img/what-we-do-in-the-shadows.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.`,
    rating: 2.5,
    director: `Claire Hodge`,
    starring: [`Amaris Hess, Tobias Osborne, Samir Fitzpatrick, Baron Bolton and other`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    reviews: [
      {
        name: `Kate Muir`,
        reviewText: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
        date: `December 24, 2016`,
        score: 2.9
      },
      {
        name: `Bill Goodykoontz`,
        reviewText: `Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        "The Grand Budapest Hotel" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.`,
        date: `November 18, 2015`,
        score: 8.2
      },
      {
        name: `Amanda Greever`,
        reviewText: `I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.`,
        date: `November 18, 2015`,
        score: 8.4
      },
      {
        name: `Matthew Lickona`,
        reviewText: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        date: `December 20, 2016`,
        score: 6.2
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.6
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.0
      },
    ]
  },
  {
    title: `Goal`,
    genre: `Comedy`,
    issuedDate: 2011,
    ratingCount: 222,
    preview: `../../markup/img/orlando.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.`,
    rating: 3.4,
    director: `Brynlee Haney`,
    starring: [`Piper Mccann, Seamus Ashley, Jude Law, Lila Waters and other`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    reviews: [
      {
        name: `Kate Muir`,
        reviewText: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
        date: `December 24, 2016`,
        score: 2.9
      },
      {
        name: `Bill Goodykoontz`,
        reviewText: `Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        "The Grand Budapest Hotel" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.`,
        date: `November 18, 2015`,
        score: 8.2
      },
      {
        name: `Amanda Greever`,
        reviewText: `I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.`,
        date: `November 18, 2015`,
        score: 8.4
      },
      {
        name: `Matthew Lickona`,
        reviewText: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        date: `December 20, 2016`,
        score: 6.2
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.6
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.0
      },
    ]
  },
  {
    title: `New Yorker`,
    genre: `History`,
    issuedDate: 2010,
    ratingCount: 442,
    preview: `../../markup/img/war-of-the-worlds.jpg`,
    description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Nunc blandit bibendum elit. Donec libero mauris, luctus sit amet rhoncus ac,
    ultrices ut elit.`,
    rating: 2.9,
    director: `Melany Short`,
    starring: [`Franklin Sosa, Abby Beasley, Jovan Lang and other`],
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    background: `img/bg-the-grand-budapest-hotel.jpg`,
    videoSrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    reviews: [
      {
        name: `Kate Muir`,
        reviewText: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious
        Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
        date: `December 24, 2016`,
        score: 2.9
      },
      {
        name: `Bill Goodykoontz`,
        reviewText: `Anderson's films are too precious for some, but for those of us willing to lose ourselves in them, they're a delight.
        "The Grand Budapest Hotel" is no different, except that he has added a hint of gravitas to the mix, improving the recipe.`,
        date: `November 18, 2015`,
        score: 8.2
      },
      {
        name: `Amanda Greever`,
        reviewText: `I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.`,
        date: `November 18, 2015`,
        score: 8.4
      },
      {
        name: `Matthew Lickona`,
        reviewText: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        date: `December 20, 2016`,
        score: 6.2
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.6
      },
      {
        name: `Paula Fleri-Soler`,
        reviewText: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        date: `December 20, 2016`,
        score: 6.0
      },
    ]
  }
];

const store = configureStore([]);

it(`Render GenresList`, () => {

  const tree = renderer
    .create(
        <Provider store={store}>
          <GenresList
            moviesList={filmsList}
            onMovieCardTitleClick={()=>{}}
          />
        </Provider>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
