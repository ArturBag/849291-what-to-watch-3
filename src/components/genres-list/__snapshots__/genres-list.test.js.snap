// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render GenresList 1`] = `
Array [
  <h2
    className="catalog__title visually-hidden"
  >
    Catalog
  </h2>,
  <ul
    className="catalog__genres-list"
  >
    <li
      className="catalog__genres-item "
    >
      <a
        className="catalog__genres-link"
        href="#"
        onClick={[Function]}
      >
        All genres
      </a>
    </li>
    <li
      className="catalog__genres-item "
    >
      <a
        className="catalog__genres-link"
        href="#"
        onClick={[Function]}
      >
        Comedies
      </a>
    </li>
    <li
      className="catalog__genres-item "
    >
      <a
        className="catalog__genres-link"
        href="#"
        onClick={[Function]}
      >
        Crime
      </a>
    </li>
    <li
      className="catalog__genres-item "
    >
      <a
        className="catalog__genres-link"
        href="#"
        onClick={[Function]}
      >
        Documentary
      </a>
    </li>
    <li
      className="catalog__genres-item "
    >
      <a
        className="catalog__genres-link"
        href="#"
        onClick={[Function]}
      >
        Dramas
      </a>
    </li>
    <li
      className="catalog__genres-item "
    >
      <a
        className="catalog__genres-link"
        href="#"
        onClick={[Function]}
      >
        Horror
      </a>
    </li>
    <li
      className="catalog__genres-item "
    >
      <a
        className="catalog__genres-link"
        href="#"
        onClick={[Function]}
      >
        Kids & Family
      </a>
    </li>
    <li
      className="catalog__genres-item "
    >
      <a
        className="catalog__genres-link"
        href="#"
        onClick={[Function]}
      >
        Romance
      </a>
    </li>
    <li
      className="catalog__genres-item "
    >
      <a
        className="catalog__genres-link"
        href="#"
        onClick={[Function]}
      >
        Sci-Fi
      </a>
    </li>
    <li
      className="catalog__genres-item "
    >
      <a
        className="catalog__genres-link"
        href="#"
        onClick={[Function]}
      >
        Thrillers
      </a>
    </li>
  </ul>,
  <div
    className="catalog__movies-list"
  >
    <article
      className="small-movie-card catalog__movies-card"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Breaking Bad"
          height={175}
          src="../../markup/img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
          onClick={[Function]}
        >
          Breaking Bad
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Pirates"
          height={175}
          src="../../markup/img/bohemian-rhapsody.jpg"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
          onClick={[Function]}
        >
          Pirates
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Misfits"
          height={175}
          src="../../markup/img/macbeth.jpg"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
          onClick={[Function]}
        >
          Misfits
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Terminal"
          height={175}
          src="../../markup/img/aviator.jpg"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
          onClick={[Function]}
        >
          Terminal
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Star Wars"
          height={175}
          src="../../markup/img/we-need-to-talk-about-kevin.jpg"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
          onClick={[Function]}
        >
          Star Wars
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Forsage"
          height={175}
          src="../../markup/img/what-we-do-in-the-shadows.jpg"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
          onClick={[Function]}
        >
          Forsage
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Goal"
          height={175}
          src="../../markup/img/orlando.jpg"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
          onClick={[Function]}
        >
          Goal
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="New Yorker"
          height={175}
          src="../../markup/img/war-of-the-worlds.jpg"
          width={280}
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
          onClick={[Function]}
        >
          New Yorker
        </a>
      </h3>
    </article>
  </div>,
  <div
    className="catalog__more"
  >
    <button
      className="catalog__button"
      type="button"
    >
      Show more
    </button>
  </div>,
]
`;
